dl/dl-base/dl-base-2.x.x.jinja

```diff
@@ -1,17 +1,9 @@
 {% extends "base.jinja" %}
 
 {% block content %}
-# TODO: merge bazel update to dl-deps
-ENV BAZEL_VERSION {{ _bazel_version }}
-
-RUN curl -LO "https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel_${BAZEL_VERSION}-linux-x86_64.deb" \
-    && dpkg --force-confnew -i bazel_*.deb \
-    && apt-get clean \
-    && rm bazel_*.deb
```

Comment:
It's still necessary but we've already installed it in `dl-deps`, the one you found here is usually a way to fix something in the upper layers of the build. 

FYI I've introduced a new one the in the TF dockerfile because the latest version of bazel is not supported.

Reply:
Sounds good, periodically we can roll up all these changes and move to the base.